Controller/TemplateController.php

```diff
@@ -135,13 +135,15 @@ public function productPricingAction(Request $request)
 
         $currencies = $this->getCurrencies($language);
         $defaultCurrency = $this->container->getParameter('sulu_product.default_currency');
+        $displayRecurringPrices = $this->container->getParameter('sulu_product.display_recurring_prices');
```

Comment:
"display" is a bit confusing here, as it does not only prevent displaying recurring prices somewhere in the frontend but disables the whole functionality.... So maybe enable_recurring_prices would be better?